src/fu-util-common.c

```diff
@@ -871,12 +871,17 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 		return NULL;
 	}
 
-	/* all devices have a name */
-	fu_common_string_append_kv (str, idt, fwupd_device_get_name (dev), NULL);
+	tmp = fwupd_device_get_name (dev);
+	if (tmp == NULL)
+		/* TRANSLATORS: Name of hardware */
+		tmp = _("Unknown Device");
```

Comment:
i think it's customary to use braces for this, even though i agree it's not a bug. With that, please merge, thanks.